conda/cli/main_clean.py

```diff
@@ -65,6 +72,66 @@ def configure_parser(sub_parsers):
     p.set_defaults(func=execute)
 
 
+# work-around for python bug on Windows prior to python 3.2
```

Comment:
Conda doesn't support python 3.2, and hasn't for a long time. Do we even need this at all?